nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -62,7 +64,7 @@
 __all__ = ['MegatronGPTPromptLearningModel']
 
 
-class MegatronGPTPromptLearningModel(MegatronBaseModel, TextGeneration):
+class MegatronGPTPromptLearningModel(MegatronBasePromptLearningModel):
```

Comment:
why drop the `TextGeneration` interface?

Reply:
The base model containers it now!

Reply:
The base class now has TextGenetation.